app/controllers/workbooks_controller.rb

```diff
@@ -0,0 +1,50 @@
+class WorkbooksController < ApplicationController
+
+  def show
+    @workbook = Workbook.find(params[:id])
+  end
+
+  def index
+    @workbooks = Workbook.all
+  end
```

Comment:
Don't you want to scope workbooks by the discipline? Does it make sense to have "all workbooks" index page?